### PR TITLE
Double beep sound added to sound synthesizer again

### PR DESCRIPTION
**Pull request by unid15**
Fixes https://github.com/d3athrow/vgstation13/issues/2900

### CODE REVIEW

File: code/game/objects/items/devices/megaphone.dm [hunk 1]
```diff
@@ -92,10 +92,10 @@ This is to cycle sounds forward
 	set category = "Object"
 	set name = "Cycle Sound Forward"
 	switch(sound_flag)
-		if(12)
+		if(13)
```
Comment (ComicIronic) [inline code/game/objects/items/devices/megaphone.dm]: DEFINES!

**Comment by ComicIronic** [comment]
I'm not sure if just naming the issue works. Try writing Fixes #whatever instead.

**Comment by Unfit** [comment]
But it's referenced here https://github.com/d3athrow/vgstation13/issues/2900 ? What would be the difference between "https://github.com/d3athrow/vgstation13/issues/2900" and "Fixes https://github.com/d3athrow/vgstation13/issues/2900"?

**Comment by ComicIronic** [comment]
One automatically closes the issue when the PR is merged, which is useful because maintainers have carpal tunnel and can only click once a day.

**Pull request merged**